sdk/maps/Azure.Maps.Timezone/src/Models/Options/TimezoneBaseOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Maps.Timezone.Models.Options
+{
+    /// <summary> Options. </summary>
+    public class TimezoneBaseOptions
+    {
+        /// <summary> Specifies the language code in which the timezone names should be returned. Please refer to <see href="https://docs.microsoft.com/azure/azure-maps/supported-languages">Supported Languages</see> </summary>
+        public string AcceptLanguage { get; set; }
+
+        /// <summary> Alternatively, use alias "o". Options available for types of information returned in the result. </summary>
```

Comment:
Do we really have the alias ("o", "s", "tf", "ty") for users? How do they use in C# SDK?